samples/matter/lock/src/combine_ota_images.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+"""
+Utility for combining Thread and Wi-Fi application binary images into one Matter OTA compliant file
+"""
+
+import sys
+import argparse
+import tempfile
+import os
+import cbor2
+import struct
+
+NCS_PATH = os.path.join(os.environ.get('ZEPHYR_BASE'), '../')
```

Comment:
I think `ZEPHYR_BASE` may not be set in some scenarios. Can we use a path relative to the current file?

Reply:
Yes, this was the initial implementation but then the script could not be run from a location different that its own (`e.g. src/combine_ota_images.py`). Maybe there is a workaround for that however.

Reply:
Hmm. So maybe let's use `ZEPHYR_BASE` if that is defined, and `__file__` otherwise?

Reply:
Ok, got it.